### ISSUE TITLE
Error: Requested MineRL from https link has different version in metadata

### ISSUE THREAD

**Issue by murakrishn** [minerllabs/minerl#583]
I am trying to install MineRL following the below command

`pip3 install --upgrade minerl`

However I am getting the following error

`ERROR: Requested minerl from https://files.pythonhosted.org/packages/40/c5/bd3413fbd181ff398c8d6aff1ee4be7be4454add377087de7d62e644af35/minerl-0.4.1.tar.gz#sha256=388b708b96a7ca2628ce436b6aa5f80e74dcdf663ca34c291b8857f8c31f8c81 has different version in metadata: '0.4.1a2'`

I am using a Mac and trying to install it within a conda environment. Any ideas on why this might be happening?

**Comment by Miffyli** [comment]
Hmm this is an interesting bug I am not able to replicate. Meanwhile could you try installing with `pip3 install git+https://github.com/minerllabs/minerl`? Current `dev` branch is more or less what the current pypi offers.

**Comment by murakrishn** [comment]
Thank you for the quick response!

The above `pip` command also failed, but the error message was more informative on this one.

It seems that I didn't have `adotopenjdk` in my path. I managed to get it running by adding the following to my `.zshrc`

```
export JAVA_PATH=/path/to/adoptopenjdk
export PATH="$PATH:JAVA_PATH/bin"
```

In my case `JAVA_PATH` was `/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home`

Once this was done, I tested out both `pip` install commands and they both work!!
